tool/inventory.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#*****************************************************************
+#*
+#* Copyright 2020 IBM Corporation
+#*
+#* Licensed under the Apache License, Version 2.0 (the "License");
+#* you may not use this file except in compliance with the License.
+#* You may obtain a copy of the License at
+
+#* http://www.apache.org/licenses/LICENSE-2.0
+#* Unless required by applicable law or agreed to in writing, software
+#* distributed under the License is distributed on an "AS IS" BASIS,
+#* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#* See the License for the specific language governing permissions and
+#* limitations under the License.
+#*
+#*****************************************************************
+# Usage:  set environment variable KAPPNAV_NAMESPACE to specify
+#         the namespace value. Default is kappnav.
+#
+echo Begin kAppNav inventory ...
+echo 
+
+# get kappnav namespace
+if [ x$KAPPNAV_NAMESPACE = 'x' ]; then
+	namespace=kappnav
+else
+	namespace=$KAPPNAV_NAMESPACE
+fi
+
+date
+server start 
+
+echo 'App Navigator resources in '$namespace' namespace:'
+echo 
+
+kubectl get -n $namespace all 
+echo 
+
+# get pods and print images 
+
+echo 'App Navigators Pods and Images:'
```

Comment:
App Navigators - singular or possessive but not plural

Reply:
Yep.  Made it singular.